addons/web/static/src/views/fields/properties/kanban_properties_field.xml

```diff
@@ -1,13 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
     <t t-name="web.KanbanPropertiesField" owl="1">
-        <div t-ref="properties" class="w-100">
+        <div t-ref="properties" class="w-100 mt-2 fw-normal text-muted">
             <div class="o_kanban_property_field d-inline-flex flex-column justify-content-center align-items-start w-100 my-1"
                 t-foreach="propertiesList"
                 t-as="propertyConfiguration"
                 t-key="propertyConfiguration.name"
-                t-if="(propertyConfiguration.value || ['integer', 'float'].includes(propertyConfiguration.type)) and propertyConfiguration.view_in_kanban">
-                <div class="mw-100 text-truncate">
+                t-if="propertyConfiguration.value and propertyConfiguration.view_in_kanban">
```

Comment:
I see this comment marked as resolved:

> Unless you have been asked to change it, I would keep it as it was before. Integers and floats may have 0 as value, which evaluate to false, but is a value and should be shown in my opinion.

But no answer to it, isn't that a real problem?

Reply:
@awa-odoo 
This issue has been resolved over task pad. The PO agreed to change it. we won't show integers and float for 0 value.